connectors/salesforce-model/src/main/java/io/syndesis/connector/salesforce/SalesforceIdentifier.java

```diff
@@ -18,13 +18,13 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-public class AnyObject {
+public class SalesforceIdentifier {
```

Comment:
Sorry got a bit confused about all those files (which are cut in GitHub, too) and I'm a bit lazy to check this out locally :). But I _thought_ that the SalesforceIdentifer comes from the mode. Why is it defined in this connector, too ?

Reply:
yeah I was lazy and moved AnyObject from one of the connectors into `salesforce-model` that's why you see rename here, there is only one, don't trust me, trust find:

```shell
[zregvart@zregvart connectors]$ find . -name SalesforceIdentifier.java
./connectors/salesforce-model/src/main/java/io/syndesis/connector/salesforce/SalesforceIdentifier.java
[zregvart@zregvart connectors]$ 
```

Reply:
sorry, thought it was still in the delete connector. all good :)